python/pip_install/private/generate_whl_library_build_bazel.bzl

```diff
@@ -232,19 +232,28 @@ def generate_whl_library_build_bazel(
         if p.startswith("@"):
             continue
 
-        os, _, cpu = p.partition("_")
+        abi, _, tail = p.partition("_")
```

Comment:
Sorry, I forget, please remind me: what's the format of these strings being parsed? Above there is a guard against "@" -- where is that coming from? The string format looks like the wheel packaging spec, but not quite?

Reply:
Added a comment, thanks for pointing this out.